### DISCUSSION TITLE
Adding new sample yaml file for setting up CI/CD with CodeMagic

### DISCUSSION THREAD

**Discussion by hoangnguyen92dn** [nimblehq/android-templates#414] [RFCs]
## Issue

As of now, we are using [CodeMagic](https://codemagic.io/start/) to execute CI/CD pipelines on some projects, there is a lack of a sample yaml file for quick setup when initializing a new project ([.cicdtemplate](https://github.com/nimblehq/android-templates/tree/develop/.cicdtemplate))

## Solution

- Create a new `.codemagic` folder
- Add new sample yaml file `codemagic.yml` and `README.md` for instruction

## Who Benefits?

Developers

## What's Next?

- [ ] Address an issue to update android-templates

**Comment by luongvo** [comment]
Voted ✅

**Comment by toby-thanathip** [comment]
Voted ✅

**Comment by chornerman** [comment]
Voted 🚀

**Comment by ryan-conway** [comment]
[Issue created](https://github.com/nimblehq/android-templates/issues/448) 🙏🏻
